core/scss/components/molecules/_ctas.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
```

Comment:
I like your point on how to name these files and that you followed the alerts pattern. However, I think that we should change and make all of these patterns singular and remove the plural names. So this file should be `cta`.

Reply:
Changed all file names and references to singular cta